app/src/main/java/org/fossasia/openevent/app/core/auth/login/LoginFragment.java

```diff
@@ -70,13 +69,24 @@ public void onStart() {
             if (!validator.validate())
                 return;
 
+            loginFragmentViewModel.getLogin().setEmail(binding.emailDropdown.getText().toString());
+            loginFragmentViewModel.getLogin().setPassword(binding.userPassword.getText().toString());
+
             ViewUtils.hideKeyboard(view);
             loginFragmentViewModel.login();
+            loginFragmentViewModel.encryption();
         });
         binding.signUpLink.setOnClickListener(view -> openSignUpPage());
         binding.forgotPasswordLink.setOnClickListener(view -> openForgotPasswordPage());
     }
 
+    @Override
+    public void onResume() {
+        super.onResume();
+        binding.emailDropdown.setText(loginFragmentViewModel.getDecryptedEmail());
+        binding.userPassword.setText(loginFragmentViewModel.getDecryptedPassword());
```

Comment:
Again, VMs responsibility to send this to view, not view's responsibility to pull

Reply:
Should livedata be used here?